src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/CreateAntigenTestProfile/CreateAntigenTestProfileViewController.swift

```diff
@@ -136,6 +142,13 @@ class CreateAntigenTestProfileViewController: UITableViewController, FooterViewH
 		return true
 	}
 	
+	func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+		return string.trimmingCharacters(in: CharacterSet.alphanumerics).isEmpty
```

Comment:
Can we have a comment here, why we did this, so new developers can understand.

Reply:
Good idea